### ISSUE TITLE
Iterate does not use Field.defaultValue/value

### ISSUE THREAD

**Issue by andlbrei** [dnbexperience/eufemia#3882]
## 🐛 Bug Report

When using `defaultValue` or `value` in `Field` within `Iterate` the value is displayed and validated, but not stored when "submitting" the contents of the `Iterate` container.

https://github.com/user-attachments/assets/fa24c62d-d910-43a0-8149-7d1508eb67e9

## To Reproduce

https://codesandbox.io/p/sandbox/iterate-defaultvalue-cdlmrl

## Expected behavior

I expect `defaultValue` and `value` to be stored when submittin to the array.  
Especially for `PushContainer`, where default values make the most sense to use.

## Eufemia Version

10.45.0

**Comment by tujoworker** [comment]
Thank you for reporting this issue 🫶
Yes, I agree, it should work. And I already tried the couple of last days to implement this, without seeing your issue (I just saw it right now).

It is tricky to get right, because of race conditions (when items adds a value, the context needs to be updated (rerender) but the iteration continues, but because of the rerender, the effect dep changes, and there we have the trouble), but I think I will give it another try, soon.

As of now, I [added a warning](https://github.com/dnbexperience/eufemia/pull/3877/files#diff-b59fb623316658e45d85d18ec433a55fa8335f037a4f85b6337bfad3c6c631e9R446), that it is not supported.
